master/pkg/model/user.go

```diff
@@ -15,6 +16,22 @@ import (
 // BCryptCost is a stopgap until we implement sane master-configuration.
 const BCryptCost = 15
 
+// ActivityType describes a user activity.
+type ActivityType string
```

Comment:
since it's a new "model" I'd put these in a new file `master/pkg/model/user_activity.go`.